sympy/solvers/tests/test_solveset.py

```diff
@@ -176,6 +176,43 @@ def test_domain_check():
     assert domain_check(0, x, oo) is False
 
 
+def test_eliminate():
+    x, y, z, t = symbols('x y z t', real=True)
+
+    assert eliminate([x+y+z], z) == S.EmptySet
```

Comment:
`x + y + z` is an expression and not an equation.
Do you mean `x + y + z = 0` ?

As a general notation, have you assumed the rhs(probably) of all the test expressions to be `0` ?